pkgs/development/mobile/androidenv/compose-android-packages.nix

```diff
@@ -2,7 +2,7 @@
 , licenseAccepted ? false
 }:
 
-{ toolsVersion ? "26.1.1"
+{ cmdLineToolsVersion ? "8.0"
```

Comment:
Since the API for this is changing, can you update the nixpkgs manual? It uses the old toolsVersion.

Reply:
Fixed on [41acb57](https://github.com/NixOS/nixpkgs/pull/208793/commits/41acb572e21fd2d1776a0e148d059a2188d68dad)